src/main/java/slimeknights/tconstruct/tables/tileentity/table/PartBuilderTileEntity.java

```diff
@@ -14,7 +14,7 @@
 public class PartBuilderTileEntity extends TableTileEntity {
 
   public PartBuilderTileEntity() {
-    super(TinkerTables.partBuilderTile.get(), "gui.tconstruct.part_builder", 4);
+    super(TinkerTables.partBuilderTile.get(), "gui.tconstruct.part_builder", 3);
```

Comment:
What is that magic number at the end? Should likely move to a constant somewhere